tests/application/test_authorization.py

```diff
@@ -188,17 +189,25 @@ def test_involuntary_authorization_decrease(
     threshold_staking.authorizationDecreaseRequested(
         staking_provider, value // 2, 0, sender=creator
```

Comment:
Let's try to avoid using `value // 2` in tests, since both the deauthorizing and remaining amount are the same and this can hide test failures.

Reply:
yep, completely agree, all new tests I used something like `value // 3`, I can prepare another PR with updating other test cases